### DISCUSSION TITLE
Level DB web admin?

### DISCUSSION THREAD

**Discussion by binarykitchen** [Level/community#139] [General]
Good evening folks,

For my web app, which is getting more traffic, I've been using `level-party` and `Level` as well. That worked well for years.
But now I feel it's too painful to investigate bugs via command line.

Wondering if there is some web admin interface to investigate each Level DB entry only administrators can access?

(I've already researched but couldn't find an answer to that)

Thanks!
M.

**Comment by binarykitchen** [comment]
Anyone??

**Reply by juliangruber** [reply]
I think this means there's none

**Reply by binarykitchen** [reply]
Indeed. Because I'm creating a web admin app to list, view edit and delete multimedia contents stored in Level DB. An existing library would have saved me time :(
